app/helpers/tags_helper.rb

```diff
@@ -1,2 +1,11 @@
 module TagsHelper
+  def goal_time_to_total(goal)
```

Comment:
this method has magic number.
Do you think about  partially using `Time` class.